DataFormats/BTauReco/src/classes_def.xml

```diff
@@ -465,4 +465,10 @@
   <class name="reco::DeepBoostedJetTagInfoRefVector"/>
   <class name="edm::Wrapper<reco::DeepBoostedJetTagInfoCollection>" persistent="false"/>
 
+  <class name="reco::PixelClusterData"/>
+  <class name="reco::PixelClusterProperties"/>
+  <class name="reco::PixelClusterTagInfo"/>
+  <class name="reco::PixelClusterTagInfoCollection"/>
```

Comment:
all non-templated classes should have a class version for a more appropriate schema evolution tracking.

Reply:
The class version (number 3) has been added to PixelClusterTagInfo and PixelClusterData.